nuxt.schema.ts

```diff
@@ -90,7 +90,8 @@ export default defineNuxtSchema({
            * @example 'https://exactpro.github.io/docs-toolkit'
            * @type {?string}
            */
-          baseUrl: undefined
+          baseUrl: undefined,
+          prefix: undefined
```

Comment:
Let's add JSDoc comment to display it during configuration.

`@type` annotation here will suggest that it can be a string.

```suggestion
          /**
           * URL prefix for cases if website is hosted not in root directory.
           *
           * @example '/docs-toolkit'
           * @type {?string}
           */
          prefix: undefined
```